sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -30,7 +35,6 @@ public async Task GetSystemTokenLiveAsync()
 
             Assert.IsNotNull(token.Token);
         }
-
         [NonParallelizable]
         [Test]
         public async Task VerifyImdsRequestMockAsync()
```

Comment:
Did you meant to remove this test as well since you moved it into ManagedIdentityClientTests?